File: src/019/problem019.py
#!/usr/bin/env python

from datetime import datetime, timedelta

date = datetime(1901, 1, 6)
sundays = 0
while date < datetime(2000, 12, 31):
	sundays += date.timetuple().tm_mday == 1
	date += timedelta(days=7)

print(sundays)

File: src/019/problem019.rb
#!/usr/bin/env ruby
require 'date'

date = Date.new(1901, 1, 6)
sundays = 0
while date < Date.new(2000, 12, 31)
    if date.mday == 1
        sundays += 1
    end
    date += 7
end

puts sundays
